### PR TITLE
Logging and Statuscake Added

### PR DESCRIPTION
**Pull request by sfawcett123**
Logging and statuscake added to the Staging Environment

### CODE REVIEW

**Comment by dfe-schoolexperience** [comment]
Review can be found at https://se-review-1779-website.azurewebsites.net

File: terraform/paas/staging.env.tfvars [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
 paas_redis_1_name         = "school-experience-staging-redis-svc"
 paas_application_name     = "school-experience-app-staging"
-paas_internet_hostnames   = ["staging-school-experience"]
+paas_internet_hostnames   = []
 application_instances     = 1
 environment               = "staging"
 azure_key_vault           = "s105d01-kv"                        # Currently Set to Development while waiting on Staging Service Now ticket to be actioned
 azure_resource_group      = "s105d01-dev-vault-resource-group"  # Currently Set to Development while waiting on Staging Service Now ticket to be actioned
+alerts = {
+  SchoolExperience_Staging = {
```
Comment (saliceti) [inline terraform/paas/staging.env.tfvars:11]: Some of these are defaults and we can simplify be removing from the resource and the config. See an example:
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/terraform/workspace_variables/production.tfvars#L22
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/terraform/modules/statuscake/main.tf#L15

File: terraform/paas/statuscake.tf [hunk 2]
```diff
@@ -0,0 +1,14 @@
+resource "statuscake_test" "alert" {
```
Comment (saliceti) [inline terraform/paas/statuscake.tf:1]: Defaults values can be removed, see other comment

File: terraform/paas/staging.env.tfvars (hunk 1 above)
Comment (sfawcett123) [reply terraform/paas/staging.env.tfvars:11]: Changed to look the same

File: terraform/paas/statuscake.tf (hunk 2 above)
Comment (sfawcett123) [reply terraform/paas/statuscake.tf:1]: Changed to be the same as above

**Comment by dfe-schoolexperience** [comment]
Review can be found at https://se-review-1779-website.azurewebsites.net

**Review by saliceti — APPROVED** [review]

**Comment by dfe-schoolexperience** [comment]
Review can be found at https://se-review-1779-website.azurewebsites.net

**Pull request merged**